jin/5주차 과제/11-선택실행구조.py

```diff
@@ -0,0 +1,73 @@
+# 세 정수 a, b, c가 입력되었을 때, 짝수만 출력
+#1
+a, b, c = map(int, input().split())
+if not a%2: print(a)
+if not b%2: print(b)
+if not c%2: print(c)
+
+#2
+a, b, c = map(int, input().split())
+print( *(filter(lambda num: num%2 == 0, [a, b, c])) )
```

Comment:
```suggestion
print( *(filter(lambda num: num%2 == 0, [a, b, c])) )
```
파이썬 내장 함수를 너무 잘 활용하신 거 같아요~